### DISCUSSION TITLE
How do I change the github account verification email address

### DISCUSSION THREAD

**Discussion by viputopia** [community/community#39917] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

github account verification email address has been blocked
How can I change the email address of GitHub account if the verification code cannot be obtained

**Comment by MitkoVtori** [comment]
I think you'll be able to change it with password and eventually phone number.

**Comment by luckwings** [comment]
Hi. I hope you can do it well.
Follow these steps.

1. In the upper-right corner of any page, click your profile photo, then click Settings.
2. In the left sidebar, click Emails.
3. If you'd like to add a new email address to set as your primary email address, under "Add email address", type a new email address and click Add.
4. Under "Primary email address", use the drop-down menu to click the email address you'd like to set as your primary email address, and click Save.

Best regards.

**Comment by airtower-luna** [comment]
If you can't log in and don't have an active session where you can change the mail address, you'll have to [contact GitHub support](https://support.github.com/request), us other users can't help there. Good luck!

**Comment by viputopia** [comment]
The situation is that i can no longer log into your github account. The email address cannot be changed from GitHub's personal Settings, nor has the mobile phone number been set to receive verification code. I tried to contact github support, but got no response.
